src/components/axes/axis.ts

```diff
@@ -54,12 +55,13 @@ module Plottable {
 
       this.formatter(formatter);
 
-      this._scale.broadcaster.registerListener(this, () => this._rescale());
+      this._rescaleCallback = (scale) => this._rescale();
```

Comment:
I personally like removing it where irrelevant as that's less to read.

There is an argument to be made that all the arguments should always be provided (less lookup), but I dunno, I value the cleanliness